Library/PhotosController.swift

```diff
@@ -48,9 +47,31 @@ class PhotosController: UICollectionViewController {
             self.sections = Photo.constructRemoteElements()
             self.collectionView?.reloadData()
         }
+
+        #if os(tvOS)
+        let playPauseTapRecognizer = UITapGestureRecognizer(target: self, action: #selector(self.playPause(gesture:)))
+        playPauseTapRecognizer.allowedPressTypes = [NSNumber(value: UIPressType.playPause.rawValue)]
```

Comment:
It's an Apple TV thingie. When you press "playPause" it starts the slideshow. You'll find this pattern everywhere when searching for how to interact with the Siri Remote.

https://developer.apple.com/library/content/documentation/General/Conceptual/AppleTV_PG/DetectingButtonPressesandGestures.html